example/src/App.tsx

```diff
@@ -270,6 +297,46 @@ const App = () => {
             }
           })()
         }
+        <AppButton
+          title={isNativeUIMRecShowing ? 'Hide Native UI MRec' : 'Show Native UI MRec'}
+          enabled={isInitialized && !isProgrammaticMRecCreated}
+          onPress={() => {
+            setIsNativeUIMRecShowing(!isNativeUIMRecShowing);
+            setIsNativeUIMRecCreated(true);
+          }}
+        />
+        {
+          (() => {
+            if (isNativeUIMRecShowing) {
+              return (
+                <AppLovinMAX.AdView adUnitId={MREC_AD_UNIT_ID} adFormat={AppLovinMAX.AdFormat.MREC} style={styles.mrec}/>
+              );
+            }
+          })()
+        }
+        <AppButton
+          title={isProgrammaticMRecShowing ? 'Hide Programmatic MRec' : 'Show Programmatic MRec'}
+          enabled={isInitialized && !isNativeUIMRecCreated}
+          onPress={() => {
+            if (isProgrammaticMRecShowing) {
+              AppLovinMAX.hideMRec(MREC_AD_UNIT_ID);
+            } else {
+
+              if (!isProgrammaticMRecCreated) {
+                AppLovinMAX.createMRec(
+                  MREC_AD_UNIT_ID,
+                  AppLovinMAX.AdViewPosition.BOTTOM_LEFT
```

Comment:
For aesthetics purposes I'd do `BOTTOM_CENTER`.

Reply:
With `BOTTOM_CENTER`, MRec covers the buttons to hide it when rendered.

Reply:
Ok `BOTTOM_LEFT` is janky. Let's do `TOP_MIDDLE` then.